app/assets/stylesheets/common/base/reviewables.scss

```diff
@@ -231,13 +231,10 @@
       font-size: var(--font-down-2);
       color: var(--secondary);
       border-radius: 8px;
-      background-color: var(--secondary-high);
       &.-flagged-post,
       &.-flagged-chat-message,
       &.-akismet-flagged-post,
-      &.-akismet-flagged-user,
-      &.-ai-flagged-post,
-      &.-ai-flagged-chat-message {
```

Comment:
All these are plugin related and should not be leaking into core stylesheets.

Reply:
Okay this I semi-know how to move them to somewhere else. Will do.

Reply:
@tgxworld do the ones for ai go here: https://github.com/discourse/discourse-ai ?
just to confirm